frame/contracts/src/tests.rs

```diff
@@ -293,6 +293,11 @@ fn compile_module<T>(wabt_module: &str)
 	Ok((wasm, code_hash))
 }
 
+fn assert_events(events: Vec<EventRecord<MetaEvent, H256>>) {
```

Comment:
should probably document somewhere that events can henceforth be inspected only after finalize() and hence we need this helper function.

Reply:
Refactored, now `System::collect_events()` will take charge of this